.github/ISSUE_TEMPLATE/crash-issues.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Crash Report
```

Comment:
Crashes are already reported to us automatically. I'm not sure this template is even required.

Reply:
there are still issues made about crashes so i would leave it

Reply:
I think if this template is ever used, the reporter can have an option to jump to this template immediately to report the bug on GitHub with an attached  Stacktrace. But that's just me.